run.php

```diff
@@ -363,15 +385,15 @@ function _argument(ReflectionParameter $p){
 	return $res;
 }
 
-function _type(ReflectionType $t){
+function _type(ReflectionType $t) : string{
 	$ret = "";
 	if($t->allowsNull()){
-	//	$ret .= "?";
+		$ret .= "?";
 	}
 	if($t->isBuiltin()){
 		$ret .= "$t";
 	}else{
-		$ret .= "\\" . ltrim($t, "\\");
+		$ret .= "\\" . ltrim($t->getName(), "\\");
```

Comment:
As of PHP 7.1, `ReflectionType->getName()` doesn't exist. I don't know why there are separated `ReflectionType` and `ReflectionNamedType`, but it needs to be investigated. Please look into it.